.github/workflows/pkgci_regression_test.yml

```diff
@@ -161,25 +221,25 @@ jobs:
             --capture=no \
             --log-cli-level=info \
             --timeout=1200 \
-            --retries 2 \
-            --retry-delay 5 \
             --durations=0 \
-            --config-files=build_tools/pkgci/external_test_suite/gpu_rocm_models_additional_flags_gfx90a.json
-      - name: "Running real weight model tests scheduled unet"
+            --config-files=${MODELS_EXTRA_FLAGS_CONFIG_FILE_PATH}
+
+      - name: "Run external tests - SDXL scheduled unet"
+        if: "matrix.sdxl-config-file != '' && !cancelled()"
         run: |
           source ${VENV_DIR}/bin/activate
           pytest SHARK-TestSuite/iree_tests/pytorch/models/sdxl-scheduled-unet-3-tank \
             -rpfE \
             -k real_weights \
             --no-skip-tests-missing-files \
-            --log-cli-level=info \
             --capture=no \
+            --log-cli-level=info \
             --timeout=1200 \
-            --retries 2 \
-            --retry-delay 5 \
             --durations=0 \
-            --config-files=build_tools/pkgci/external_test_suite/sdxl_scheduled_unet_gpu_rocm_gfx90a.json
+            --config-files=${SDXL_CONFIG_FILE_PATH}
+
       - name: "Running SDXL rocm pipeline benchmark"
+        if: contains(matrix.name, 'rocm')
```

Comment:
want to add the same success check here or I can do it in my PR too

Reply:
I'm not quite sure how much I want to include other conditions yet, so leaving that off.